src/endpoint/endpoint.js

```diff
@@ -190,6 +192,28 @@ async function main(options = {}) {
             }));
         }
 
+        fs.watch(ssl_utils.get_cert_dir('S3'), {},
+            (event_type, filename) => {
+                ssl_utils.update_certs_from_disk().then(is_updated => {
```

Comment:
Any reason not to use async/await? We are trying to avoid using promise chains.
would also prefer this function to be outside of fs.watch. maybe with cert name as a parameter as I will need this code also for EXTERNAL_DB certificates

Reply:
When I try to use await instead of then I get that await can only be use in top level.

The new version has this new functionality inside ssl_utils.